tracing/redis.go

```diff
@@ -0,0 +1,60 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v7"
```

Comment:
I wonder if there are cases where we want to allow for use of https://github.com/gomodule/redigo, too.

Reply:
I think baseplate.go should pick and stick with a single third party redis library.